src/sdl2_wrapper.cpp

```diff
@@ -615,6 +615,7 @@ CAMLprim value Val_SDL_Event(SDL_Event *event) {
     break;
   case SDL_KEYDOWN:
   case SDL_KEYUP:
+    fprintf(stderr, "Got keyup event\n");
```

Comment:
Is this intentionally left in?

Reply:
Whoops, nope. Was debugging events not getting passed, turns out Revery somehow pulls in an old version of esy-sdl2 without some patches that make pan events work